### ISSUE TITLE
Where's the command help?

### ISSUE THREAD

**Issue by trusktr** [BrowserSync/browser-sync#1929]
running `browser-sync start --help` shows barely anything, I don't see any options (like `--port`):

```
❯ npx browser-sync start --help 
browser-sync start

Start the server

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
```

And this doesn't show the options either:

```
❯ npx browser-sync --help
browser-sync [command]

Commands:
  browser-sync start   Start the server
  browser-sync init    Create a configuration file
  browser-sync reload  Send a reload event over HTTP protocol
  browser-sync recipe  Generate the files for a recipe

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]

For help running a certain command, type <command> --help
  browser-sync start --help

You can run a static server by providing a path(s) directly
  browser-sync app/src app/tmp

If the directory contains a 'index.html' file, you can omit any input
  browser-sync

You can run the proxy in this manner too
  browser-sync https://example.com

To run a proxy, whilst also serving static files
  browser-sync https://example.com htdocs/themes/example
```

**Comment by trusktr** [comment]
In particular, I wanted to look up the format of `--watch` and the website doesn't say.

**Comment by shakyShane** [comment]
yeah this looks like a bug, I'll get to it shortly :)

**Comment by shakyShane** [comment]
fixed in `browser-sync@2.27.9` - thank you for the report :)
